### ISSUE TITLE
Comments on Twitter comments?

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#106]
Yesterday I added a new feature on Scripting News that allows you to directly RT a post from Scripting News on Twitter. What do you think? Will you use it? Does it give you other ideas?

**Comment by danderson3** [comment]
yes.  I often have kudos to offer and links to share in response

**Comment by scripting** [comment]
The next question is how to gather the comments together, other than the #scriptingnews tag in Twitter. The problem for me is that I have to remember to look for the comments, rather than having them come to me. That suggests there should also be an RSS feed of the comments.

**Comment by andysylvester** [comment]
I had a problem using the comment feature yesterday. I was not logged into Twitter at the time. When I clicked on the link, I got a prompt to log into Twitter. The focus then returned to the Scripting News page, and a blank dialog box appeared, along with a dialog which seemed to say it was having problems connecting with Twitter. However, the text was blocked by the blank dialog box. This morning, I decided to log into Twitter on another tab, then switch to Scripting News and attempt to comment. This time, it worked. The blank dialog box from yesterday now appeared as a text box referring to the Scripting News post, and allowed me to post. Just wanted to pass this on....
